packages/components/src/components/post-header/post-header.tsx

```diff
@@ -193,9 +198,6 @@ export class PostHeader {
       this.mobileMenuAnimation.finish(); // no animation
     }
 
-    const mhh = this.host.shadowRoot.querySelector('.local-header')?.clientHeight;
```

Comment:
This should remain in the `handleResize` as well otherwise the value is not updated when you go from desktop to mobile and vice-versa.